_raml/apps/products/examples/patch_product_productid_request.json

```diff
@@ -31,6 +31,40 @@
                 ]
             }]
 },
+{
+  "op": "add",
```

Comment:
@mhoehn Since this should represent a real example request it doesn't really make sense to add all three operation options because it's just not a realistic scenario. So I would choose only one to represent. Furthermore, the one, that is chosen, has to be reflected accordingly in the example respond to this request.

Reply:
I'm really struggling with putting all operations in the request cause it doesn't make sense. Nobody would use a request like that because in the end it doesn't change anything, if I'm not mistaken. But I see why you would like to present all options to make it clearer. So let's go with your solution to make it as easy as possible for external developers.